zebrad/src/commands/start/sync.rs

```diff
@@ -0,0 +1,208 @@
+#![allow(unused_variables, dead_code)]
```

Comment:
anything in particular we're leaving unused?

Reply:
this was just to reduce noise while I was setting everything up, will remove